config/400-controller.yaml

```diff
@@ -41,6 +41,24 @@ spec:
           imagePullPolicy: Always
           ports:
             - containerPort: 8080
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /live
+              port: 8080
+              scheme: HTTP
+            periodSeconds: 15
+            successThreshold: 1
+            timeoutSeconds: 1
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /live
+              port: 8080
+              scheme: HTTP
+            periodSeconds: 15
+            successThreshold: 1
+            timeoutSeconds: 1
```

Comment:
may be better to have 5

Reply:
> from the issues the hub team has maybe 1 is too low

in case of hub there is a db ping which is taking more than 1 sec due to which it is failing
in pac case it just return "ok" 😄

Reply:
do we wanna increase?